packages/ui/src/dropdown/DropdownTrigger.tsx

```diff
@@ -3,6 +3,9 @@ import { Placement } from 'popper.js';
 import React, { Component, MouseEventHandler, ReactNode } from 'react';
 import { Manager, Popper, Reference } from 'react-popper';
 
+export const CHECKOUT_ROOT_NODE_ID = 'checkout-app';
```

Comment:
I believe we don't need export these consts

Reply:
Would be better to refactor bcapp part so we could have same Id for local work and for the build 😆